.github/workflows/search-client-master.yaml

```diff
@@ -0,0 +1,24 @@
+name: search-client-master
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - medicines/search-client-master/**
+      - .github/workflows/search-client-master.yaml
+
+jobs:
+  build-and-test:
+    name: Build, Test and Deploy
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone Repo
+        uses: actions/checkout@v2
+        with:
+          path: products
+
+      - name: Test
+        working-directory: ./medicines/search-client
+        run: cargo test
```

Comment:
`search-client-branch.yaml` and `search-client-master.yaml` do the same thing so we could potentially just have one of them

Reply:
True, just matches the pattern of the other packages and allows it to diverge.